.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Keep GitHub Actions up to date with GitHub's Dependabot... dependabot.yml
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    groups:
+      github-actions:
+        patterns:
+          - "*"  # Group all Actions updates into a single larger pull request
```

Comment:
We usually don't do this as one update could cause things to fail and then the other updates can't be merged.

BTW, this is our sort of default dependabot config: https://github.com/exercism/prolog/blob/main/.github/dependabot.yml

Reply:
Why is the default config not running on this repo?